app/models/request.rb

```diff
@@ -7,6 +7,7 @@ class Request < ApplicationRecord
   validates :content, :user, :duedate, presence: true
 
   enum status: { pending: 0, accepted: 1, rejected: 2, commented: 3 }
+  enum open_closed: { open: 0, closed: 1 }
```

Comment:
It means when request accepted, it's automatically closed, right?
Before, I thought user close requests by click 'close' button, but I think automatic is better.
So I also changed the tabs 'Open / To close / Closed' to 'Open / Closed'